cylc/rose/entry_points.py

```diff
@@ -71,8 +71,7 @@ def post_install(srcdir=None, opts=None, rundir=None):
         srcdir=srcdir, opts=opts, rundir=rundir
     )
     # Finally dump a log of the rose-conf in its final state.
-    if results['fileinstall']:
-        dump_rose_log(rundir=rundir, node=results['fileinstall'])
+    dump_rose_log(rundir=rundir, node=results['fileinstall'])
```

Comment:
Without type checking I'm not confident in this change. It looks to me like the `node` argument shouldn't be a `False` or `None` so I think the truthiness test was valid?

Reply:
https://github.com/cylc/cylc-rose/pull/196/files#r1033649794

Reply:
https://github.com/cylc/cylc-rose/blob/2a68e4c1438400e81fd44dad2a84bbff70d46630/cylc/rose/entry_points.py#L227-L237

Reply:
This has been hanging around too long. A quick re-read suggests you are right.

```suggestion
    if results['fileinstall']:
        dump_rose_log(rundir=rundir, node=results['fileinstall'])
```